docs/application/tizen-studio/setup/prerequisites.md

```diff
@@ -24,6 +24,8 @@ Follow these instructions to install the appropriate JDK version for your system
 
 You can also use Oracle's OpenJDK. For more installation details, see [OpenJDK 12 and OpenJFX Installation Guide](openjdk.md).
 
+> **NOTE** You must use Oracle Java Development Kit (JDK) 8 only to use RT IDE.
```

Comment:
Please follow the note style in the comment. The indentation must be checked also.


  \> \*\*Note**
  \>
  \> For RT IDE use Oracle Java Development Kit (JDK) 8 only.

And this update should be placed above line 25, not after line 25.

Reply:
done